plugins/ingress-nginx.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ingress-nginx
+spec:
+  shortDescription: Interact with ingress-nginx
```

Comment:
We recently added a `homepage` field, too.
Do you mind pointing it to the plugin documentation?

Reply:
Once https://github.com/kubernetes/ingress-nginx/pull/3982 is merged, I'll add a link to the docs.